### PR TITLE
Replace 'AREA' by using 'area'

### PR DESCRIPTION
**Pull request by hsiangheleellnl**
Replace 'AREA' in the code by using 'area'. Note: 'AREA' is 2-D (time, ncol) but 'area' is only 1-D (ncol).

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = chemdyg
-version = 0.1.2
+version = 0.1.3
```
Comment (tangq) [inline setup.cfg]: The version update should be done in a separate PR.

**Review by tangq — CHANGES_REQUESTED** [review]

**Comment by tangq** [comment]
Have you tested this PR with the latest coupled simulation v3alpha02? If the test is successful, I will approve it.
https://acme-climate.atlassian.net/wiki/spaces/CM/pages/3849715728/20230704.v3alpha02.historical+0101.chrysalis

File: setup.cfg (hunk 1 above)
Comment (hsiangheleellnl) [reply setup.cfg]: Correct back to v0.1.2

**Review by tangq — APPROVED** [review]
